lib/rollbax/item.ex

```diff
@@ -27,7 +27,7 @@ defmodule Rollbax.Item do
     if map_size(meta) == 0 do
       data
     else
-      Map.put(data, "custom", meta)
+      Map.merge(data, meta)
```

Comment:
Actually we can go third option which seems to solve both issues: make a special key in `meta`, and merge it with `data`. There is an example of similar thing in the Plug: https://github.com/elixir-lang/plug/blob/6fbfc865aa6750f67a1c60aa1e834cba3510c59a/lib/plug/adapters/cowboy.ex#L33.